lib/Gocdb_Services/ServiceService.php

```diff
@@ -459,6 +459,35 @@ public function getServiceTypes() {
         return $types;
     }
 
+    /*
+     * Validates the the 'production => monitored' rule for the user inputted 
+     * service data.
+
+     * @param array $serviceValues
+     * @throws \Exception If the serviceValues production/monitored combination
+     * is invalid. The \Exception's message will contain a human readable error
+     * message.
+     * @return null
+     */
+    private function validateProductionMonitoredCombination($serviceValues) {
```

Comment:
Good to have this factored out. I guess this'll make using a config value easier.